[2025-02-14T10:25:57.885+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T10:25:57.920+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: new_dag.push_message scheduled__2025-02-13T00:00:00+00:00 [queued]>
[2025-02-14T10:25:57.938+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: new_dag.push_message scheduled__2025-02-13T00:00:00+00:00 [queued]>
[2025-02-14T10:25:57.941+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-14T10:25:58.313+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): push_message> on 2025-02-13 00:00:00+00:00
[2025-02-14T10:25:58.328+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=125) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T10:25:58.332+0000] {standard_task_runner.py:72} INFO - Started process 148 to run task
[2025-02-14T10:25:58.332+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'new_dag', 'push_message', 'scheduled__2025-02-13T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/new_dag.py', '--cfg-path', '/tmp/tmpnxebloxh']
[2025-02-14T10:25:58.335+0000] {standard_task_runner.py:105} INFO - Job 35: Subtask push_message
[2025-02-14T10:25:58.414+0000] {task_command.py:467} INFO - Running <TaskInstance: new_dag.push_message scheduled__2025-02-13T00:00:00+00:00 [running]> on host 7b78e6c4b0cc
[2025-02-14T10:25:58.524+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='new_dag' AIRFLOW_CTX_TASK_ID='push_message' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-13T00:00:00+00:00'
[2025-02-14T10:25:58.527+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-14T10:25:58.528+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-14T10:25:58.529+0000] {logging_mixin.py:190} INFO - Current task name:push_message state:running start_date:2025-02-14 10:25:57.922278+00:00
[2025-02-14T10:25:58.529+0000] {logging_mixin.py:190} INFO - Dag name:new_dag and current dag run status:running
[2025-02-14T10:25:58.530+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-14T10:25:58.554+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-14T10:25:58.567+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-14T10:25:58.569+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=new_dag, task_id=push_message, run_id=scheduled__2025-02-13T00:00:00+00:00, execution_date=20250213T000000, start_date=20250214T102557, end_date=20250214T102558
[2025-02-14T10:25:58.606+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-14T10:25:58.607+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-14T10:25:58.608+0000] {logging_mixin.py:190} INFO - Dag name:new_dag queued_at:2025-02-14 10:12:23.025627+00:00
[2025-02-14T10:25:58.609+0000] {logging_mixin.py:190} INFO - Task hostname:7b78e6c4b0cc operator:PythonOperator
[2025-02-14T10:25:58.630+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T10:25:58.655+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-14T10:25:58.660+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
